applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -4,3 +4,281 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#if (CONFIG_AUDIO_DEV == HEADSET)
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/audio/audio.h>
+
+#include "le_audio.h"
+#include "macros_common.h"
+#include "ctrl_events.h"
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/audio/audio.h>
+#include <bluetooth/audio/capabilities.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(cis_headset, CONFIG_LOG_BLE_LEVEL);
+
+static le_audio_receive_cb receive_cb;
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+};
+
+#define CHANNEL_COUNT_1 BIT(0)
+static struct bt_codec lc3_codec =
+	BT_CODEC_LC3(BT_CODEC_LC3_FREQ_ANY, BT_CODEC_LC3_DURATION_10, CHANNEL_COUNT_1, 40u, 120u,
+		     1u, (BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL | BT_AUDIO_CONTEXT_TYPE_MEDIA),
+		     BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
+
+static struct bt_conn *default_conn;
+static struct bt_audio_stream streams;
+static struct bt_audio_stream *stream_cap_config_cb(struct bt_conn *conn,
```

Comment:
This is a static function, so should be separated from the struct declarations

Reply:
Git plugin show something which made me though there's a blank line :(